src/openforms/js/lang/nl.json

```diff
@@ -2063,7 +2063,7 @@
     "originalDefault": "Configure"
   },
   "hxssl8": {
-    "defaultMessage": "Objects API integration",
+    "defaultMessage": "Integratie met de Objecten API",
```

Comment:
Hmm, this one is similar to "verouderde instellingen", I like to place the most important words first in this case: "Objects API", leading to "Objecten API integratie"?